instance/outputs.tf

```diff
@@ -1,47 +1,77 @@
 output "instance_ids" {
-  value = ["${concat(aws_instance.instance.*.id,aws_instance.instance_no_ebs.*.id)}"]
+  value = concat(
```

Comment:
See higher, with TF 0.12 we should simplify the module using dynamic blocks, so we don't need these workarounds anymore.